src/main/java/com/hibernate/cinema/model/dto/CinemaHallResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package com.hibernate.cinema.model.dto;
+
+public class CinemaHallResponseDto {
+    private long id;
```

Comment:
```suggestion
    private Long id;
```
default value of long = 0L;